src/components/OTP.vue

```diff
@@ -309,6 +309,14 @@ export default {
     /** The user type is set as soon as component is created */
     this.userType = this.groupData.userType;
   },
+  mounted() {
+    /** If user already logged in, get from store and redirect to destination */
+    let phoneNumber = this.$store.getters.getUserPhoneNumber;
```

Comment:
Instead of using store as `this.$store.getters...` you should use `mapState` and `mapGetters`
you can mix the store variables into your own list of data variables and computed variables of a vue component and then you can use it directly like `this.userPhoneNumber`

Read more here https://next.vuex.vuejs.org/guide/state.html#the-mapstate-helper